### ISSUE TITLE
Performance of Flower102, cifa100, in VTAB

### ISSUE THREAD

**Issue by xben-cherry** [FightingFighting/GPS#10]
I have recently been trying to reproduce the results from VTAB, but the results for the Flower102 task in VTAB differ significantly from the data presented in the paper. The result I reproduced is 87.7, and I saw in another issue thread that you mentioned using the old training script, which yielded a result of 99.5. I also achieved a result of 68.7 on the CIFAR-100 dataset, while the paper reports a result of 81.1. Could you please let me know which training script that is? I would also appreciate it if you could provide the specific training parameters, such as learning rate, batch size, etc. Thank you very much!

**Comment by FightingFighting** [comment]
As for cifar100, in order to compare with VPT, we run GPS on their code. Maybe you could try to implement our method in their code. For the flower, it is kind of weird, as I tried to rerun the code several weeks ago, and it can achieve 99+% on accuracy. Have you got any wrong for the code? 

All the scripts are new and you can use them to run. The hyperparameters of them are provided in the training scripts.

**Comment by xben-cherry** [comment]
Thank you for your reply. I'm sure that I run the code you provided with the hyperparameters in the training scripts. Is  the flower102 you reproduce in FGVC rather than VTAB? I achieve good performance in FGVC but not in VTAB. Could you give me some advice please?

**Comment by xben-cherry** [comment]
I have solved the problem.  The codes you provided have somethings wrong. The datasets Flowers102 and pets in dataset_factory.py are loaded in ImageDataset, rather than VTAB. Change the _VTAB_DATASET 'oxford_iiit_pet' to 'pets' and 'oxford_flowers102' to 'flowers102'.

**Comment by FightingFighting** [comment]
thank you very much!!!

**Comment by Jwonno** [comment]
Hi @xben-cherry , Were you able to successfully reproduce the performance on the CIFAR-100 dataset? If so, did you use the same hyperparameters as in the provided train script? 
Thank you!

**Comment by xben-cherry** [comment]
@Jwonno I used the same hyperparameters in train script but I couldn't get the same performance. The author said the cifar100 was reproduced on VPT code. But I didn't attempt.

**Comment by Jwonno** [comment]
@xben-cherry Thank you for answer!
